msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -408,6 +408,95 @@ In the stage view, users can zoom in and zoom out of the image. You can select t
 > * Zoom in and zoom out capability applies only to the image elements that is image type in an Adaptive Card.
 > * For Teams mobile apps, stage view functionality for images in Adaptive Cards is available by default. Users can view Adaptive Card images in stage view by simply tapping on the image, irrespective of whether the `allowExpand` attribute is present or not.
 
+### Overflow menu on Adaptive Cards 
+
+Overflow menu is a new feature added to Adaptive Cards, you are allowed to select up to six primary buttons that appears on Adaptive Cards, if there are more than six, which is secondary set of buttons will be placed inside the overflow menu.
+
+**There are two types of overflow Menu:**
+
+* **Actions**: In Adaptive Card, overflow menu is placed after primary set of buttons, secondary buttons are inside the overflow menu.
+
+**Following image is an example for actions**
+
+   ![action card](~/assets/images/Cards/action-image-revised.png)
+
+* **Action set** is defined as an action inside the action set, where you can perform multiple actions on an Adaptive Card and each action set can have an overflow menu.
+
+**Following image is an example for action set**
+
+   ![action set card](~/assets/images/Cards/action-set-image.png)
+
+**What happens when the user selects overflow menu?**
+
+**Desktop**: on a desktop when you select the overflow menu, whatever you have effectively set as secondary buttons will appear in the display.
```

Comment:
replace:
whatever you have effectively set as secondary appears.

Reply:
changes are done